programmers/난이도별/level01.완주하지_못한_선수/yoojinhee03.java

```diff
@@ -0,0 +1,18 @@
+import java.util.Arrays;
+class Solution {
+    public String solution(String[] participant, String[] completion) {
+      
+        	String answer="";
+       	Arrays.sort(participant);
+	Arrays.sort(completion);
+	int i;
+	for(i=0; i<completion.length; i++) {
+		if(!completion[i].equals(participant[i])) {
+			answer=participant[i];
+			break;

```

Comment:
```suggestion
			return participant[i];
```
여기서 바로 return 해도 괜찮을 것  같아요!